modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -79,15 +109,26 @@ and it must be loaded before the SDK.
 
 // end::install[]
 
-== Hello Couchbase
 
-Couchbase uses xref:7.0@server:learn:security/roles.adoc[Role Based Access Control (RBAC)] to control access to resources.
+== Cloud Connections
```

Comment:
This section now unnecessary provided 
1. kv timeout extended in code sample
2. troubleshooting given at end -
see https://docs-staging.couchbase.com/python-sdk/current/hello-world/start-using-sdk.html#troubleshooting for details.